pkg/utils/utils.go

```diff
@@ -0,0 +1,55 @@
+package utils
+
+import (
+	"os"
+
+	"github.com/containers/image/pkg/sysregistries"
+	"github.com/containers/image/signature"
+	"github.com/sirupsen/logrus"
+)
+
+// CheckConfigFiles log messages if one or more files to set up buildah do not exist
+func CheckConfigFiles() {
+	registriesConfPath := sysregistries.RegistriesConfPath(nil)
+	_, err := os.Stat(registriesConfPath)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Errorf("Required file registries.conf not found in %s, %s", registriesConfPath, err)
+		} else {
+			logrus.Errorf("error checking registries.conf in %s, %s", registriesConfPath, err)
+		}
+	}
+
+	mountsDefaultFilePath := "/usr/share/containers/mounts.conf"
+	mountsOptionalFilePath := "/etc/containers/mounts.conf"
+	_, errDefault := os.Stat(mountsDefaultFilePath)
+	_, errOptional := os.Stat(mountsOptionalFilePath)
+	if errDefault != nil && errOptional != nil {
+		if os.IsNotExist(errDefault) && os.IsNotExist(errOptional) {
+			logrus.Errorf("Required file mounts.conf not found in %s or %s.", mountsDefaultFilePath, mountsOptionalFilePath)
+		} else {
+			logrus.Errorf("error checking mounts.conf in %s and %s, %s, %s", mountsDefaultFilePath, mountsOptionalFilePath, errDefault, errOptional)
+		}
+	}
+
+	seccompDefaultPath := "/usr/share/containers/seccomp.json"
+	seccompOverridePath := "/etc/crio/seccomp.json"
+	_, errDefault = os.Stat(seccompDefaultPath)
+	_, errOptional = os.Stat(seccompOverridePath)
+	if errDefault != nil && errOptional != nil {
+		if os.IsNotExist(errDefault) && os.IsNotExist(errOptional) {
+			logrus.Errorf("Required file seccomp.json not found in %s or %s, please ensure that the containers-common package has been installed.", seccompDefaultPath, seccompOverridePath)
+		} else {
+			logrus.Errorf("error checking seccomp.json in %s and %s, %s, %s", seccompDefaultPath, seccompOverridePath, errDefault, errOptional)
+		}
+	}
+
+	_, err = signature.DefaultPolicy(nil)
+	if err != nil {
+		if os.IsNotExist(err) {
+			logrus.Errorf("Required file policy.json not found %s", err)
+		} else {
+			logrus.Errorf("error checking policy.json, %s", err)
```

Comment:
You need to print out the path here.

/etc/containers/policy.json Which I think signature.DefaultPolicy will return?